### PR TITLE
Port utils module to pytest

### PR DESCRIPTION
**Pull request by utkbansal**
Ths is a copy of PR 1419, have cleaned up the history here.

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

**Comment by utkbansal** [comment]
@kain88-de See this is what I was saying - I don't even understand how am I getting commits from you and Oliver.

**Comment by utkbansal** [comment]
Woah, I squashed them into @richardjgowers's commit and it all disappeared.

**Comment by utkbansal** [comment]
Haha, even quantified code is confused now 😆

**Comment by utkbansal** [comment]
Something very fishy here, those commits and diff disappeared into thin air. 
Or maybe I still don't completely understand git..

**Comment by jbarnoud** [comment]
Can you detail the steps you followed?

**Comment by utkbansal** [comment]
Sure

* I first squashed Richard's commits into one commit, Max & Oliver's commits into a single commit and my work into 2 commits
* Then I squashed Max & Oliver's commit into Richard's commit
* Then I rebased to remove the merge conflict

**Comment by utkbansal** [comment]
What I didn't understand was why was I seeing commits from Max & Oliver on doing `git rebase -i HEAD~x` and not while doing `git log --oneline`.

File: .travis.yml [hunk 1]
```diff
@@ -23,16 +23,16 @@ env:
     - PYTHON_VERSION=2.7
     - COVERALLS=false
     - NOSE_FLAGS="--processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50"
-    - NOSE_TEST_LIST="analysis auxiliary core topology utils"
+    - NOSE_TEST_LIST="analysis auxiliary coordinates core topology"
```
Comment (richardjgowers) [inline .travis.yml]: this would revert the coordinates to use nose

**Comment by utkbansal** [comment]
This should be good to be merged now!

**Pull request merged**